src/js/modules/articles/reducer.js

```diff
@@ -2,123 +2,83 @@ import {
-      volume: 108,
-      issue: 1,
-      date: "July 19, 2017",
-      time: "8:27 AM",
-      sectionSlug: "creative-writing",
-    },
-  },
+  articles: {},
 };
 
 const reducer = (state = { ...initialState }, action) => {
   switch (action.type) {
     case FETCH_ARTICLE_PENDING: {
-      return { ...state, isFetching: true };
-    }
-    case FETCH_ARTICLE_REJECTED: {
-      return { ...state, isFetching: false, error: action.payload };
+      return { ...state, isFetching: true, };
     }
     case FETCH_ARTICLE_FULFILLED: {
       return {
         ...state,
         isFetching: false,
         isFetched: true,
-        request: action.payload,
+        // action.payload already tested in actions to be array
+        articles: action.payload.reduce((acc, cur) => {
+          const articleSlug = cur.slug;
+          cur.dateline = formatDate(cur.created_at);
+          cur.sectionSlug = slugFinder(cur.section_id, action);
+          delete cur.section_id;
+          acc[ articleSlug ] = cur;
+          return acc;

```

Comment:
You can remove the assignments of `cur.dateline` and `cur.sectionSlug` with the use of a spread operator. So you would keep the `articleSlug` declaration and the `section_id` deletion, and then you would do:
```
acc[ articleSlug ] = {
    ....cur,
    dateline: FORMAT DATE,
    sectionSlug: FIND SLUG
}
```